### DISCUSSION TITLE
UI/Editor

### DISCUSSION THREAD

**Discussion by Barugon** [bevyengine/bevy#4488] [Q&A] [answered]
Now that 0.7 has been released, will there finally be some movement on UI and an editor?

**Comment by Weibye** [comment]
I know its a late reply but the current status of this is _yes_ there is movement but its not going to be very visible to the end user just yet. Our current work is preparing to transition from `stretch` (no longer maintained layout library) to `taffy` (our fork of `stretch`). See this: #4716 

We are still building core units that we can build on top of, and this is one of them. If you or anyone else wants to help getting the editor-work started sooner, please consider heading over to `taffy` and help us build that :) 

https://github.com/DioxusLabs/taffy/issues

There's quite a few tasks that should be fairly low effort or complexity.

**Reply by Barugon** [reply]
Thanks for the reply.

**Comment by Barugon** [comment]
0.8 is out. Is UI design still paralyzed?

**Reply by Nilirad** [reply]
We adopted a train release schedule right after 0.6, so features are no longer blocked on a release.

As you can see from both GitHub issues/PRs and on the Discord server, in the `#ui-dev` channel, there has been quite some movement in the UI area lately.

EDIT: Even the [release notes] mention UI improvements for the next release.

[release notes]: https://bevyengine.org/news/bevy-0-8/#what-s-next

**Reply by Barugon** [reply]
I missed the blurb in the release notes.

**Comment by Weibye** [comment, marked as answer]
> 0.8 is out. Is UI design still paralyzed?

No! 😄 We've started working on building the UI and as @Nilirad state, we are no longer _blocked_ (and we kind of never were after 0.6). The biggest change now that 0.8 is out is that we're spending some effort on the project management side of things to make sure we remove a lot of the _uncertainty_ of what to focus on at any given time and to not be blocked on the review side of things.

You can find the [UI Project board here](https://github.com/orgs/bevyengine/projects/10) but expect both the project board's contents and how we work with it to change over time.

The plan is to expand on what we have and make `bevy_ui` a robust and ergonomic UI framework that feels native to the ECS experience but there's lot of work to get there so if you are reading this, consider joining our efforts in the `#ui-dev` channel in the Bevy discord.

**Reply by Barugon** [reply]
Awesome. Thanks for the link.
